docs/user-guide.md

```diff
@@ -98,10 +98,12 @@ for i in *.md; do grep -r $i . | grep -v "^\./$i" > /dev/null; rv=$?; if [[ $rv
   Provides resource information such as size, type, and quantity to assist in
   assigning Kubernetes resources appropriately.
 
+* **CLI roadmap** ([cli-roadmap.md](cli-roadmap.md)): Kubernetes
```

Comment:
@bgrant0607 , maybe I can move cli-roadmap to devel subdir? and link it in docs/devel/README.md?

I see that we also have roadmap.md in the user guide, but this roadmap looks fine (for me).